packages/core/package.json

```diff
@@ -5,7 +5,7 @@
 	"main": "dist/index.js",
 	"sideEffects": false,
 	"scripts": {
-		"test": "eslint src --ext ts,tsx && jest",
+		"test": "eslint src --ext ts,tsx && jest && istanbul-badges-readme --silent",
```

Comment:
This is very cool!

I get an error `Error: Coverage file does not exist in ./coverage/coverage-summary.json`, I guess because no test coverage is being generated.

Also: The README of the root package currently gets copied to the core package when publishing. This is so both on the Remotion Github Repo and on the NPM package you can see the README.

So I am unsure if the test coverage of the core package should be at the top of the README. I would prefer to keep it clean and selectively only put the most important information in the README. However happy to put it somewhere else if this is possible.

Reply:
As discussed, we moved everything to `TESTING.md`. @calvin-f has written a script so we have zero dependencies for the badge generation👌🏻  Makes a lot more sense now!